app/build.gradle.kts

```diff
@@ -122,6 +122,7 @@ dependencies {
     coreLibraryDesugaring(libs.tools.desugar.jdk)
 
     implementation(libs.blurView)
+    implementation(libs.androidx.connect.client)
```

Comment:
Can we change this to `fullImplementation` as the sensor is only in the full flavor?

Reply:
Realised that the code to check permissions is in main package so reverted the change.

Reply:
Also didn't realize that. There are no GMS dependencies inside it thankfully so all good!